src/containers/RecipeInputPage/css/index.css

```diff
@@ -0,0 +1,141 @@
+.body {
+  background-color: white;
+}
+
+.main {
```

Comment:
we shouldn't be using classnames like body or main, they're too generic and we're prompt to collision as I realize now that I also named 'main' one of my classes